packages/p2p/src/components/modal-manager/modals/filter-modal/filter-modal-header.jsx

```diff
@@ -2,7 +2,7 @@ import * as React from 'react';
 import { observer } from 'mobx-react-lite';
 import { localize, Localize } from 'Components/i18next';
 import { useStores } from 'Stores';
-import PageReturn from 'Components/page-return/page-return.jsx';
+import PageReturn from 'Components/page-return';
```

Comment:
refactor of these sections not done. can be taken up when working on these files.